### PR TITLE
build: Add markdownlint config file

### PR DESCRIPTION
**Pull request by abmusse**
Add config to address https://github.com/IBM/ibmi-oss-docs/issues/50#issuecomment-727230576 and https://github.com/IBM/ibmi-oss-docs/pull/52/files#r524382604

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "MD013": {
+     "code_blocks": false,
+     "headings": false
+  },
+  "MD026": {
+     "punctuation": ".,;:!。，；：！"
+  },
+  "no-trailing-spaces": false
```
Comment (kadler) [inline .markdownlint.json:9]: Is there any way to limit this to code blocks? I don't really want to allow trailing spaces elsewhere.

Comment (abmusse) [reply .markdownlint.json:9]: Looking at the rule [MD009](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md009---trailing-spaces) I don't see a parameter to set for code_blocks.

Comment (kadler) [reply .markdownlint.json:9]: That's disappointing. In that case, I guess we should just leave that off and we'll have to strip trailing spaces from code blocks.

Comment (abmusse) [reply .markdownlint.json]: Yeah that it disappointing.

**Review by kadler — APPROVED** [review]

**Pull request merged**